cmd/tmpfs/tmpfs.go

```diff
@@ -0,0 +1,347 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"log"
+	"net"
+	"os"
+	"sync"
+
+	"github.com/Harvey-OS/go/internal/tmpfs"
+	"github.com/Harvey-OS/ninep/pkg/debugfs"
+	"github.com/Harvey-OS/ninep/protocol"
+)
+
+var (
+	networktype = flag.String("ntype", "tcp4", "Default network type")
+	netaddr     = flag.String("addr", ":5641", "Network address")
+	debug       = flag.Int("debug", 0, "Print debug messages")
+	pkg         = flag.String("package", "", "tar.gz package to open")
+)
+
+type fileServer struct {
+	Versioned bool
+
+	archive    *tmpfs.Archive
+	filesMutex sync.Mutex
```

Comment:
lots of times people use a shorter name, like, I kid you not, m, or ... just sync.Mutex and then you do
f.Lock()
f.Unlock()
That's a ken thing also in his c ompiler btw.